client/src/components/Charts/Charts.jsx

```diff
@@ -83,15 +89,26 @@ class Charts extends React.Component {
       parent_group: this.state.drilldown, // see: note in totals.js regarding ignoring this property
       group: elems[0]._chart.config.data.labels[elems[0]._index],
       // time: this.props.time
-      state: 'all', // default, overriden if state appears
-      // published: false //default
+      state: null, // one of the following will be changed and then filled into lockItem
+      county: null,
+      // published: false // filled in by filters, used to filter data once retrieved from backend
+      lockItem: null,
+      filterLevel: this.props.lockType,
     };
-    this.props.filters.forEach((f) => params[f[0]] = f[1]);
-
+    this.props.filters.forEach((f) => params[f[0]] = f[1]); 
+  
+    if  (!params.state && !params.county) {
+      params.lockItem = 'all';
+    } else if (!params.state) {
+      params.lockItem = params.county;
+    } else if (!params.county) {
+      params.lockItem = params.state;
+    }
+  
```

Comment:
If possible, leave out specializing logic up to its parent. Line 98 should be generic enough for the parent(s) to set filters to run.